pkg/client/deploymentconfigs.go

```diff
@@ -107,3 +110,17 @@ func (c *deploymentConfigs) Rollback(config *deployapi.DeploymentConfigRollback)
 		Into(result)
 	return
 }
+
+// Get returns information about a particular deploymentConfig
+func (c *deploymentConfigs) GetScale(name string) (result *extensions.Scale, err error) {
+	result = &extensions.Scale{}
+	err = c.r.Get().Namespace(c.ns).Resource("deploymentConfigs").Name(name).SubResource("scale").Do().Into(result)
+	return
+}
+
+// Update updates an existing deploymentConfig
+func (c *deploymentConfigs) UpdateScale(scale *extensions.Scale) (result *extensions.Scale, err error) {
+	result = &extensions.Scale{}
+	err = c.r.Put().Namespace(c.ns).Resource("deploymentConfigs").Name(scale.Name).SubResource("scale").Body(scale).Do().Into(result)
```

Comment:
Well, it appears it currently attempts with the Origin v1 Codec, which won't work for us...